Tools/sitl_run.sh

```diff
@@ -128,16 +128,25 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 		SIM_PID=$!
 
 		# Check all paths in ${GAZEBO_MODEL_PATH} for specified model
-		readarray -d : -t paths <<< ${GAZEBO_MODEL_PATH}
-		for possibleModelPath in "${paths[@]}"; do
+		IFS=":"
```

Comment:
You typically backup the original value of IFS (`IFS_bak=$IFS`) and then restore it after the loop to avoid surprises in the rest of the script.

Reply:
@bkueng Makes sense! Addressed in the last commit!